openstack/nova/templates/_console-service.yaml.tpl

```diff
@@ -9,6 +9,7 @@ metadata:
   name: nova-console-{{ $name }}
   annotations:
   {{- include "utils.topology.service_topology_mode" . | indent 2 }}
+  {{- include "utils.linkerd.pod_and_service_annotation" . | indent 4 }}
```

Comment:
There is a mismatch... Is that really correct?

Reply:
Yes, this come from the fact that `utils.linkerd.pod_and_service_annotation` doesn't contain any indentation while `utils.topology.service_topology_mode` [contains 2 spaces](https://github.com/sapcc/helm-charts/blob/5904aaefdd8bcdd7ab04babe8a2e4a60adf1a069/openstack/utils/templates/_topology.tpl#L69).

Reply:
Ah, my bad.